Add safety planning (#52)
eclipse-score/score-crates: docs/verification_report/module_verification_report.rst

**1.3. on Detailed Design**
       - List of detailed design tags tested by which test case, passed/failed and completeness verdict
         (this shall be generated by tools and accompanied by progress charts to be usable also for project steering)
       - This is split in a list of QM components tested and a separate list of tests for ASIL rated components.
       - List of detailed design tags (ASIL rated) linked to inspection checklist and verdict (derived from PR export)

       - The lists may also contain other verification methods like "Analysis" - process tbd

**2. DFA Report**
       - List of the performed component DFA, pass/fail with open mitigations

**3. Safety Analysis Report**
       - List of the performed component Safety Analysis, pass/fail with open mitigations

**4. Unit Verification Coverage**

**4.1. Structural Coverage**
       `coverage report <https://github.com/eclipse-score/score-crates/pull/31#issue-4254036938>`_

**4.2. Static Code Analysis**
       - List of the units with compiler warning numbers and coding rule violation numbers

>          (this shall be generated by tools and accompanied by progress charts to be usable also for project steering)

Reviewer: ```suggestion
         (this shall be generated by clippy linting tool)
```
